src/components/reportPage/BatchSubmitPage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./batch-job.css";
+
+const BatchSubmitPage = () => {
+  return (
+    <div className="section_container">
+      <h1 style={{ textAlign: "center" }}>Batch Job Submitted Successfully!</h1>
+      <p style={{ textAlign: "center", fontSize: 20 }}>Your results will be sent to your email. </p>
```

Comment:
I think its best to show their email here(`something@domian.com`), this is because most people have more than one email, they might forget which email they entered when they press submit

Reply:
It would be useful for them to confirm that they entered their email correctly. No reason why this can't be done later though

Reply:
I've added the option to change email after submitting the job.
![Email option after submit](https://user-images.githubusercontent.com/81655881/131275927-ec11f9b4-8834-4fce-b54c-cc472dd7e5c9.PNG)

Reply:
This is a great feature, but you need to discuss this with the backend team, what if the user changed their email after the job completed (the email was sent ), what happens then?

Reply:
Good initiative, but this feature probably isn't gonna be implemented from the backend side

Reply:
Good point, haven't thought of that. I think it would be a good idea to use a multistep form with a progress bar on top since there is a sequence of actions for this job. 
Step 1: Upload files, the uploaded files would have had pages to scroll through, similar to Gmail where you can scroll between older and newer emails. 
Step: 2: Enter email
Step 3: Confirmation screen, they can review their email and files uploaded for the final time before submitting the job to backend. 
I will create a design on Figma, also will remove this change email option and simply display their email for now.